docs/reference/alias.asciidoc

```diff
@@ -5,9 +5,8 @@
 An alias is a secondary name for a group of data streams or indices. Most {es}
 APIs accept an alias in place of a data stream or index name.
 
-You can change the data streams or indices of an alias at any time. If you use
-aliases in your application's {es} requests, you can reindex data with no
-downtime or changes to your app's code.
+If you use aliases in your application's {es} requests instead of directly using indices, you can update or change the backing data with **no
```

Comment:
```suggestion
Using aliases in your application's {es} requests allows you to update or change backing data with no downtime or code changes. This allows production systems to instantly switch between data streams or indices of an alias, without interruption.
```

Reply:
(I edited my edit, it's getting very meta here)

Reply:
Reindexing is key use case for aliases